app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -35,10 +34,16 @@
         android:hint="@string/fractions_hint_text"
         android:inputType="text"
         android:longClickable="false"
+        android:textSize="16sp"
         android:maxLength="200"
-        android:padding="8dp"
+        android:minHeight="48dp"
+        android:paddingStart="16dp"
+        android:paddingLeft="16dp"
```

Comment:
You haven't defined end padding over here.

Reply:
done

Reply:
Done